peerpod-ctrl/Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN CC=gcc CGO_ENABLED=${CGO_ENABLED} GOOS=${TARGETOS:-linux} GOARCH=${TARGETARC
 FROM --platform=$TARGETPLATFORM registry.fedoraproject.org/fedora:38
 ARG CGO_ENABLED=1
 
-RUN if [ "$CGO_ENABLED" = 1 ] ; then dnf install -y libvirt-libs && dnf clean all; fi
+RUN if [ "$CGO_ENABLED" = 1 ] ; then dnf install -y libvirt-libs /usr/bin/ssh && dnf clean all; fi
```

Comment:
nit: Would it better to use the ssh package name eg `openssh-clients` instead of the absolute path from a portability standpoint?